docs/New Game.md

```diff
@@ -152,3 +152,28 @@ misconfiguration in Randovania.
 Before a game can work in Multiworld, it must first work properly as a solo game. It is highly recommended to finish the basic integration first.
 
 **TODO**
+
+# Expectations of a Development Game
```

Comment:
I would move this section further up.
Either above Multiworld, or above Adding a new game entry.

Reply:
When reading the document, it feels weird to go over all this text about after the game is added, before I can read how to add a game.

Reply:
it's why i proposed "above MW" as the other moving location. (Realistically, games will probably never launch with MW for stable)

For me what is also a bit weird is that all this info is very much helpful for deciding on when/if you want to upstream/add a game at all.

Reply:
agreed with moving it above MW